### ISSUE TITLE
The new calendar icon for .ics file download needs tooltip / title

### ISSUE THREAD

**Issue by kivinen** [ietf-tools/datatracker#3695]
### What happened?

The new icon on the http://localhost:8000/wg/ipsecme/meetings/ page for downloading .ics file is so "abstract" that it needs tooltip to explain what it is. It looks like monitor with title bar on the top, but I think it is supposed to be some kind of calendar... The old icon was bit more understandable, but we should add a check that you can't use any kind icons without tooltip text that explains what it does.

For example on the page http://localhost:8000/meeting/113/session/ipsecme we do have multiple similar icons, but all of them (including the download calendar file icon) do have tooltips which explain what they do.

I was wondering can we make a test that goes through all the html templates we have and verify that any of them using any of the icons do actually also include tooltip?

### Code of Conduct

- [X] I agree to follow the IETF's Code of Conduct

**Comment by larseggert** [comment]
Pick a different one from https://icons.getbootstrap.com/?

**Comment by kivinen** [comment]
Lars Eggert writes:
> Pick a different one from https://icons.getbootstrap.com/?

I think better fix is to add tooltip, as almost every other icon
already has. For example this same icon on the agenda page etc do have
tooltip, so why not add one here too. That would solve the problem
properly.

Even if I pick another icon, there are probably going to be someone
who complains that specific icon is not good, so changing the icon
does not really help (now when I know what the icon is supposed to be,
I can understand it, but when I first encountered it it looked more
like screen icon that calendar icon, and as both of them would have
worked in the location (i.e., calendar vs meetecho link) I was
confused until I noticed that link points to the url ending in .ics).
-- 
***@***.***
